### ISSUE TITLE
[new design] All page titles say "Welcome"

### ISSUE THREAD

**Issue by kgodey** [creativecommons/creativecommons.github.io-source#447]
Page titles should reflect the current page (I mean the title in the `<title>` tag).

**Comment by Dhruvi16** [comment]
I am not sure how to fix this because it is the way it should be.

**Comment by kgodey** [comment]
@Dhruvi16 I am not sure what you mean by "it is the way it should be".

On the current site, if you go to https://opensource.creativecommons.org/community/, the page title in the `<title>` tag matches the actual page title ("Join the Developer Community")

On the new site, the page title just says "Welcome" https://cc-open-source-dev.netlify.app/community/

**Comment by Dhruvi16** [comment]
Yes yes, I recognized the problem, and by "it is the way it should be" I meant the code for the title. And I will have to look it up more to find the solution for this.

**Comment by kgodey** [comment]
@Dhruvi16 no worries, I wasn't sure if I had been clear :)

**Comment by Aakash2408** [comment]
> Page titles should reflect the current page (I mean the title in the `<title>` tag).

    Hey, @kgodey I understand the issue can I take this?

**Comment by Dhruvi16** [comment]
Hey, @Aakash2408 I have already assigned you an issue. And this issue is related to my internship work so it will be better if I handle it. Thanks for the interest though.

**Comment by Aakash2408** [comment]
> Hey, @Aakash2408 I have already assigned you an issue. And this issue is related to my internship work so it will be better if I handle it. Thanks for the interest though.

Hey @Dhruvi16 , No problem 😃.

**Comment by Dhruvi16** [comment]
Okay I know I did a horrible mistake of removing title blocks from the templates.
